### DISCUSSION TITLE
feature for email app

### DISCUSSION THREAD

**Discussion by dinhokusanagi** [linuxdeepin/developer-center#4670] [Features Request & Ideas | 特性请求 & 头脑风暴]
the deepin email application does not have the possibility of starting minimized along with the system in the taskbar, every time the system is started and the application opens I have to close it for it to stay in the bar.  I've tried several commands in the .config folder without success. I've concluded that it's an unavailable feature, this is a very useful feature and I'd like it to be added if possible.

**Comment by BLumia** [comment]
cc @meiyixiang
